templates/cloudflare-workers/worker/worker.ts

```diff
@@ -0,0 +1,37 @@
+import { AutoRouter, cors, error, IRequest } from 'itty-router'
+import { handleAssetDownload, handleAssetUpload } from './assetUploads'
+import { handleUnfurlUrl } from './bookmarkUnfurling'
+import { Environment } from './types'
+
+// make sure our sync durable object is made available to cloudflare
+export { TldrawDurableObject } from './TldrawDurableObject'
+
+// we use itty-router (https://itty.dev/) to handle routing. in this example we turn on CORS because
+// we're hosting the worker separately to the client. you should restrict this to your own domain.
+const { preflight, corsify } = cors({ origin: '*' })
```

Comment:
```suggestion
// we're hosting the worker separately to the client. you should restrict this to your own domains.
const { preflight, corsify } = cors({ origin: ['*'] })
```

might be a tiny bit easier for folks to adapt if it's already an array.

Reply:
errr nvm it's not the same behaviour in an array